webui/react/src/components/UserSelectFilter.tsx

```diff
@@ -57,8 +54,8 @@ const UserSelectFilter: React.FC<Props> = ({ onChange, value }: Props) => {
       );
     }
 
-    if (users) {
-      const allOtherUsers = users
+    if (Loadable.isLoaded(users)) {
```

Comment:
You're not actually handling the loading state here, we're still just showing an empty Select.